nautobot/core/tests/test_utils.py

```diff
@@ -276,6 +278,53 @@ def test_list_of_strings(self):
 class GetFooForModelTest(TestCase):
     """Tests for the various `get_foo_for_model()` functions."""
 
+    @classmethod
+    def setUpTestData(cls):
+        # Define some simple models dynamically for testing
+        class UserModelTest(Model):  # noqa: DJ008 test model, no __str__ needed
+            user = ForeignKey(settings.AUTH_USER_MODEL, blank=True, null=True, on_delete=CASCADE)
+
+            class Meta:
+                app_label = "tests"
+
+        class CreatorModelTest(Model):  # noqa: DJ008 test model, no __str__ needed
+            creator = ForeignKey(settings.AUTH_USER_MODEL, on_delete=CASCADE)
+
+            class Meta:
+                app_label = "tests"
+
+        class NoUserModelTest(Model):  # noqa: DJ008 test model, no __str__ needed
+            name = CharField(max_length=50)
```

Comment:
Maybe to make the test stronger, actually call this field `user` to make sure it doesn't cause an error?


```suggestion
            user = CharField(max_length=50)
```